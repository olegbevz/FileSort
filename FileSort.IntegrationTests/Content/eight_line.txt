423778. Banana Banana Cherry Something Banana Something
355795. Apple Something
958936. Banana Apple the
564947. best the Cherry is Cherry Apple Something Apple the
353740. Banana Banana best Cherry is Banana best best
852069. Apple is Banana best
858830. Banana the best Banana Something Apple
337629. Apple best
