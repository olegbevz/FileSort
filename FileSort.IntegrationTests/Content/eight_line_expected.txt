337629. Apple best
852069. Apple is Banana best
355795. Apple Something
958936. Banana Apple the
353740. Banana Banana best Cherry is Banana best best
423778. Banana Banana Cherry Something Banana Something
858830. Banana the best Banana Something Apple
564947. best the Cherry is Cherry Apple Something Apple the
